File: javascript/1684.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function (allowed, words) {
  let ans = 0;
  words.forEach((word) => {
    let res = word;
    for (let i = 0; i < allowed.length; i++) {
      res = res.replaceAll(allowed[i], "");
    }
    if (res == "") {
      ans += 1;
    }
  });
  return ans;
};

File: go/1684.go
package leetcode

import "strings"

func countConsistentStrings(allowed string, words []string) int {
	ans := 0
	for _, val := range words {
		res := val
		for _, allow := range allowed {
			res = strings.ReplaceAll(res, string(allow), "")
		}
		if res == "" {
			ans += 1
		}
	}
	return ans
}